### PR TITLE
Pcq 514 pcq without case new api

### PR DESCRIPTION
**Pull request by ManojKulkarni-HMCTS**
### JIRA link (if applicable) ###
PCQ-514

### Change description ###
A new API - pcqRecordWithoutCase - has been introduced. This API is based on the pcqWithoutCase but will now return service id and actor information. The existing APIs are unchanged. The change also includes unit tests, integration test and functional test.
In addition, a false positive for OWASP dependency check has been suppressed.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/pcqbackend/controllers/GetPcqRecordWithoutCaseTest.java [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    public void getPcqRecordWithoutCaseMultipleIds() {
+        try {
+
+            //Create the Test Data 3 times in the database.
+            String jsonStringRequest = jsonStringFromFile(JSON_FILE);
+            PcqAnswerRequest answerRequest = jsonObjectFromString(jsonStringRequest);
+            answerRequest.setCompletedDate(updateCompletedDate(answerRequest.getCompletedDate()));
+            pcqBackEndClient.createPcqAnswer(answerRequest);
+
+            PcqAnswerRequest answerRequest2 = cloneAnswerObject(answerRequest,"INTEG-TEST-11", "PROBATE1",
```
Comment (davejones74) [inline src/integrationTest/java/uk/gov/hmcts/reform/pcqbackend/controllers/GetPcqRecordWithoutCaseTest.java:179]: petitioner (divorce) and applicant (probate) wrong way round?

File: src/main/java/uk/gov/hmcts/reform/pcqbackend/utils/ConversionUtil.java [hunk 2]
```diff
@@ -228,4 +250,12 @@ public static String validateRequestHeader(List<String> requestHeaders) throws I
 
     }
 
+    private static PcqAnswerResponse createPcqRecord(ProtectedCharacteristics pcqDbRecord) {
```
Comment (davejones74) [inline src/main/java/uk/gov/hmcts/reform/pcqbackend/utils/ConversionUtil.java:253]: Could this method name be more specific? I.e. createConsolidationServicePcqRecord? Not that it matters, it's a private method so can be changed later if more return types are required.

File: src/integrationTest/java/uk/gov/hmcts/reform/pcqbackend/controllers/GetPcqRecordWithoutCaseTest.java (hunk 1 above)
Comment (ManojKulkarni-HMCTS) [reply src/integrationTest/java/uk/gov/hmcts/reform/pcqbackend/controllers/GetPcqRecordWithoutCaseTest.java:179]: Done

File: src/main/java/uk/gov/hmcts/reform/pcqbackend/utils/ConversionUtil.java (hunk 2 above)
Comment (ManojKulkarni-HMCTS) [reply src/main/java/uk/gov/hmcts/reform/pcqbackend/utils/ConversionUtil.java:253]: Done

**Pull request merged**